config/pulumi/Pulumi.stackname.yaml.example

```diff
@@ -293,7 +293,7 @@ config:
   ############################################################################
   # This is the Kubernetes version to install using Digital Ocean K8s. Changing this value may result in bugs in the
   # behavior of the reference architecture because it is designed around the APIs for 1.19.
-  domk8s:k8s_version: 1.21.11-do.0
+  domk8s:k8s_version: 1.22.8-do.0
```

Comment:
**Comment**: This version change (and the previous version) are a little incongruous with the comment above about MARA being unpredictable outside version 1.19.  Perhaps we should update the comment to be amenable to future updates?

**Question**:  What is the relationship between the Digital ocean version and the versions of K8S?  Are their minor updates generally important?

Reply:
You're right - the 1.19 comment is old, so I'm going to update that before merge. We had initially pinned things to 1.19 back way back when we started because there was a big change on the horizon with the Ingress Objects (the API changed versions), but without that in the way I am inclined to just try and build with latest and if it passes great. If not, we debug. 

To your K8 question with DO. They (DO) seem to be the only ones who restrict versions like this; with everyone else you pick a version and it kinda stays there (albeit with adoption to update). DO has their own specific -do versions and they seem to have two - 1.21.x and 1.22.x going on, and they seem to be updated frequently. My plan is to test with latest and just let it always pick that so we don't have to keep changing this, but I do want to test.